android/expoview/src/main/java/host/exp/exponent/taskManager/AppRecordInterface.java

```diff
@@ -1,4 +1,4 @@
-package expo.loaders.provider.interfaces;
+package host.exp.exponent.taskManager;
```

Comment:
Not needed, but logical. Since I've changed AppLoader to AppStarter, and there is no notion of AppRecordInterface in it, this interface would just stay here for no purpose.

Now, AppRecordInterface is used only internally, in expo client and managed workflow. There is no need to expose this interface.